### DISCUSSION TITLE
using d X and Y location to generate buffer arround these points

### DISCUSSION THREAD

**Discussion by GDDS2022** [orbisgis/geoclimate#808] [Q&A]
Hello,

can I configure the .json config file with a list of x and y (lat long) in order to process a list of areas arround these points?

regards,

**Comment by j3r3m1** [comment]
Hi,
Currently it is possible to replace a list of zones that need to be calculated by a list of buffer around points. For that you need to use groovy scripts. However, you need to keep in mind that the calculations will:
1. create TSU within each of your buffer
2. calculate the indicators for each building, blocks and TSU
3. calculate the LCZ if asked for each TSU

This means that you will not have a single building density, LCZ, SVF, etc. for each of your buffer but rather building density, LCZ, SVF, etc. for each RSU of each of your buffer and I suppose this is not what you want. Am I right ?

**Reply by GDDS2022** [reply]
Well, I'lll had to group the attribute by localisation and make some spatial analysis using a buffer on GIS to gather and merge indicatros for each X Y point, Of course It will be easier to have One value per buffer directly,

but how do I write the line one the config file to specify X, Y and buffer? 

"locations":[["X","Y","BUFFER"],["X2","Y2","BUFFER"],["X3","Y3","BUFFER"],["X4","Y4","BUFFER"]],

**Comment by GDDS2022** [comment]
Well, I'lll had to group the attribute by localisation and make some spatial analysis using a buffer on GIS to gather and merge indicatros for each X Y point,

but how do I write the line one the config file to specify X, Y and buffer?

**Comment by j3r3m1** [comment]
Some of the indicators cannot be merged simply since they cannot be averaged or summed simply (I mean you can but they would get wrong). This is why I initially suggested you to use a several steps approach:
1. create a buffer around your stations
2. use the full GeoClimate chain on the entire zone of interest
3. intersect and merge the RSU included within each station buffer
4. rerun the calculations for each merged geometries (use the "computeAllGeoIndicators" using your table containing the merged geometries as "rsuTable" input.

If however, you prefer using a method where you aggregate the UCP parameters of all RSU contained within a given buffer tell us and @ebocher might give you more information about how to use Groovy to make the calculation for zones being buffer that you design.
